### DISCUSSION TITLE
Can fit_pseudolabel be used for text predictors?

### DISCUSSION THREAD

**Discussion by Wang-Lin-boop** [autogluon/autogluon#2081] [General]
I find the `tabularpredictor` has many useful features, which include `fit_pseudolabel`, `refit_full`, `distill`, `fit_summary`, `leaderboard` and `feature_importance_analysis`, could I use them on the `textpredictor`?

**Comment by sxjscience** [comment]
- `distill` is available in TextPredictor. See example: https://github.com/awslabs/autogluon/tree/master/examples/automm/distillation
- For `fit_pseudolabel`, how would you use it?
- For `refit_full`, this is something that we may support in the future. But it is not clear how to do it correctly for deep networks.
- For `fit_summary`, TextPredictor will actually create a tensorboard file for you to track the learning results.
